12_MINI/error-codes.yaml

```diff
@@ -397,11 +381,7 @@ Errors:
   text: ""
   id: "HEATBREAK_THERMISTOR_FAIL"
   type: "CONNECT"
-- code: "12822"
-  title: ""
-  text: ""
-  id: "NOZZLE_DOES_NOT_HAVE_ROUND_SECTION"
-  type: "CONNECT"
+# 12822 reserved NOZZLE_DOES_NOT_HAVE_ROUND_SECTION
```

Comment:
Does it mean that mini can't have square nozzles, or does it mean that mini is just not able to detect them?

Reply:
This error can occur in tool offset calibration while touching a round object on the bed to calibrate it, so it is XL only thing